plugins/woocommerce/src/Blocks/Patterns/PTKPatternsStore.php

```diff
@@ -176,7 +187,7 @@ public function fetch_patterns_on_plugin_update( $upgrader_object, $options ) {
 	 * @return void
 	 */
 	public function flush_cached_patterns() {
-		delete_transient( self::TRANSIENT_NAME );
+		delete_option( self::OPTION_NAME );
```

Comment:
I think we should also delete the `last_fetch_patterns_request` option. So if the tracking is disabled accidentally then activated again right after, the patterns can be fetched right away without waiting for another day.

Reply:
Good point! Fixed with a200530

Reply:
Thanks for the update! LGTM! The failed metric workflow seems flaky.